apps/meteor/server/models/raw/Sessions.ts

```diff
@@ -1616,4 +1604,25 @@ export class SessionsRaw extends BaseRaw<ISession> implements ISessionsModel {
 
 		return this.col.bulkWrite(ops, { ordered: false });
 	}
+
+	async updateDailySessionById(_id: ISession['_id'], record: Partial<ISession>): Promise<UpdateResult> {
+		return this.updateOne({ _id }, { $set: record }, { upsert: true });
+	}
+
+	async updateAllSessionsByDateToComputed(from: Date, to: Date): Promise<UpdateResult | Document> {
+		return this.updateMany(
+			{
+				type: 'session',
+
+				// we filter by createdAt to be able to perform a range query, since the "day", "month" and "year" fields were not meant for that
+				createdAt: { $gte: from, $lt: to },
+			},
```

Comment:
Do we need an index on type+createdAt? Not sure how the collection data is composed